gtsam/hybrid/HybridFactorGraph.cpp

```diff
@@ -22,6 +22,8 @@ using namespace std;
 
 namespace gtsam {
 
+using GFG = GaussianFactorGraph;
```

Comment:
Let’s just use the type. I know I did this, but now  I think it will be more readable if we don’t typedef